test/virt/dual/topology.dot.in

```diff
@@ -7,7 +7,8 @@ graph "dual" {
         node [shape=record, fontname="monospace"];
 	edge [color="cornflowerblue", penwidth="2"];
 
-	qn_template="infix-x86_64";
+	qn_template="infix-bios-x86_64";
+	qn_image="infix-x86_64-disk-test.img"
```

Comment:
I would suggest that you squash in the changes to `qn_image` into the next commit, as the file you are referencing does not yet exist.

It is always good to have the mindset that each individual commit should build and and be usable on its own. Being able to use `git bisect` with minimal hassle can save lots of time when trying to pinpoint when a regression was introduced, for example.

You could then rephrase this commit message to describe that you're aligning the `dual` topology to use the same template as the `quad`.

Reply:
Sure, thnx. Will do that.